jobs/kpi-forecasting/kpi_forecasting.py

```diff
@@ -11,22 +25,157 @@
 }
 
 
+class KPIPipeline:
+    def __init__(self, config_path):
+        self.config_data = load_yaml(filepath=config_path)
+        if isinstance(self.config_data["forecast_model"]["parameters"], list):
+            self.model_type = "funnel"
+            self.model_class = FunnelForecast
+            self.segments = list(self.config_data["metric_hub"]["segments"].keys())
+        else:
+            self.model_type = "prophet"
+            self.model_class = ProphetForecast
+            self.segments = None
```

Comment:
This logic seems quite brittle; it doesn't really leave room for a forecast model type that has a list of parameters and is _not_ FunnelForecast. Why not leave this as a parameter in the config?

Reply:
Yeah that's fair, added model_type back to the configs and updated this logic